pom.xml

```diff
@@ -40,28 +38,21 @@
   </developers>
 
   <scm>
-    <connection>scm:git:git://github.com/jenkinsci/dashboard-view-plugin.git</connection>
-    <developerConnection>scm:git:git@github.com:jenkinsci/dashboard-view-plugin.git</developerConnection>
-    <url>https://github.com/jenkinsci/dashboard-view-plugin</url>
-    <tag>dashboard-view-2.9.8</tag>
```

Comment:
A non-`HEAD` `tag` in `master` indicates some kind of mistake using the release plugin, possibly just using an obsolete version.